Leanplum/Cache/LPCache.h

```diff
@@ -7,14 +7,17 @@
 //
 
 #import <Foundation/Foundation.h>
-#import "LPCaching.h"
+#import "LPRegion.h"
+#import "LPFileAttribute.h"
 
-NS_ASSUME_NONNULL_BEGIN
-
-@interface LPCache : NSObject<LPCaching>
+@interface LPCache : NSObject
```

Comment:
Is there a reason to move away from protocol and tie into one particular implementation here?

Reply:
This is a good question. The caching protocol was only used by LPCache. Usually protocols are valuable when you want to enforce a behavior across multiple classes. I do not forsee LPCaching protocol to be implemented by any other class. Thats the reason its removed.

Reply:
Protocols are valuable when there may be different ways to do the same thing. The idea here was to have a caching protocol independent of implementation and then have a specific implementation (in this case, Keychain) conform to it. This gives us the flexibility to change to a different backing storage (like File) if needed

Reply:
Sure. i can bring back the protocol.

Reply:
protocol has been added.